source/common/json/config_schemas.cc

```diff
@@ -19,7 +19,8 @@ const std::string Json::Schema::LISTENER_SCHEMA(R"EOF(
               "type" : "string"
             }
           },
-          "cipher_suites" : {"type" : "string"}
+          "cipher_suites" : {"type" : "string"},
```

Comment:
Q: From schema perspective, should we verify non-empty? Or do we want to allow empty string to mean no ECDH? What makes sense?